### PR TITLE
Remove Clearcut dependency.

### PR DESCRIPTION
**Pull request by jeremyjiang-dev**

### CODE REVIEW

**Review by paulb777 — COMMENTED** [review]
Can the GTMLogger references be removed from FPRClient.m and the podspec?

File: FirebasePerformance/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Version 7.3.1(Unreleased)
+* Deprecate Clearcut event transport mechanism.
+
+# Version 7.3.0
+* Add blocklist for GoogleDataTransport upload URLs.
+* Update the event transport mechanism to open sourced GoogleDataTransport.
```
[the remark this answers is not in the document]
Comment (jeremyjiang-dev) [reply FirebasePerformance/CHANGELOG.md]: Fixed.

File: FirebasePerformance/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+# Version 7.3.1(Unreleased)
```
[the remark this answers is not in the document]
Comment (jeremyjiang-dev) [reply FirebasePerformance/CHANGELOG.md]: Thanks for spotting this! We have not decided when to release yet. I will remove the version number for now and add them once we decide when to release.

**Comment by visumickey** [comment]
> Can the GTMLogger references be removed from FPRClient.m and the podspec?

This is something we definitely would want to do. But this would be on a separate PR and not the intent of this PR.

File: FirebasePerformance/Sources/FPRClient+Private.h [hunk 3]
```diff
@@ -45,17 +45,17 @@
...
 
 /**
- * Cleans up the log directory path in the cache directory created for Clearcut logs storage.
+ * Cleans up the log directory path in the cache directory created for Fll logs storage.
  *
  * @remark This method (cleanup logic) should stay for a while until all of our apps have migrated
  * to a version which includes this logic.
  */
-+ (void)cleanupClearcutCacheDirectory;
++ (void)cleanupFllCacheDirectory;
```
[the remark this answers is not in the document]
Comment (jeremyjiang-dev) [reply FirebasePerformance/Sources/FPRClient+Private.h]: I wonder if we should remove these clearcut cleanup methods?

[the remark this answers is not in the document]
Comment (jeremyjiang-dev) [reply FirebasePerformance/Sources/FPRClient+Private.h]: Discussed offline. We need to keep `cleanupClearcutCacheDirectory` to remove Clearcut cache directory for users who just update to the FLL only SDK.

File: FirebasePerformance/Sources/FPRClient.m [hunk 4]
```diff
@@ -252,9 +252,9 @@ - (void)processAndLogEvent:(FPRMSGPerfMetric *)event {
       }];
 }
 
-#pragma mark - Clearcut log directory removal methods
+#pragma mark - FLL log directory removal methods
```
[the remark this answers is not in the document]
Comment (jeremyjiang-dev) [reply FirebasePerformance/Sources/FPRClient.m]: Fixed.

File: FirebasePerformance/Sources/FPRClient+Private.h [hunk 5]
```diff
@@ -45,17 +45,17 @@
 /**
  * Determines the log directory path in the caches directory.
  *
- * @return The directory in which Clearcut logs are stored.
+ * @return The directory in which Fll logs are stored.
```
[the remark this answers is not in the document]
Comment (jeremyjiang-dev) [reply FirebasePerformance/Sources/FPRClient+Private.h]: Fixed.

**Comment by visumickey** [comment]
@jeremyjiang-dev Can we resurrect this PR to remove the clearcut dependency from our iOS code?

**Comment by paulb777** [comment]
Please rebase or merge with master.

**Comment by jeremyjiang-dev** [comment]
> @jeremyjiang-dev Can we resurrect this PR to remove the clearcut dependency from our iOS code?

Resolved the conflict and requested for review.

**Comment by jeremyjiang-dev** [comment]
> Please rebase or merge with master.

Done.

**Review by paulb777 — APPROVED** [review]

**Comment by visumickey** [comment]
> Can the GTMLogger references be removed from FPRClient.m and the podspec?

This is already done in a different PR.

**Review by visumickey — APPROVED** [review]

**Pull request merged**